plugins/modules/zos_copy.py

```diff
@@ -2014,8 +2015,10 @@ def allocate_destination_data_set(
     # empty dataset was created for the user by an admin/operator, and they don't have permissions
     # to create new datasets.
     # These rules assume that source and destination types are compatible.
+    # Ensure if the dest_exist pass and empty dict
```

Comment:
Could you explain this comment a bit more? It's a little confusing to me at the moment.